### DISCUSSION TITLE
Communication through GitHub

### DISCUSSION THREAD

**Discussion by tokjun** [SmartNeedle/SystemIntegration#2] [Announcements]
Hi there-

I activated the "Discussions" feature in our GitHub repositories to see if we can use it to discuss details. Topics related to specific components should be done in individual repos. We can use the "SmartNeedle/SystemIntegration" repo as a place to discuss overall system design.

**Comment by nobyhata** [comment]
this is useful. Shall we try this?

**Comment by lisaepfl** [comment]
@dlezcan1 What command do you now use to launch system bringup? I have tried removing the needleParamFile argument but I get a launching error.

**Reply by dlezcan1** [reply]
You still need to use needleParamFile argument. The launch arguments that we’re removed were numCH and numAA. needleParamFile is how we load in the needle’s calibration matrices.

**Reply by lisaepfl** [reply]
I see, thank you very much!
